guides/common/modules/proc_configuring-a-bmc-interface.adoc

```diff
@@ -30,15 +29,21 @@ ifndef::satellite[]
 . On the *Proxies* tab, select your {SmartProxy} as *BMC Proxy*.
 endif::[]
 . Click *Submit*.
-. Navigate to *Hosts* > *All Hosts*.
-. Click *Edit* next to the host you want to edit.
-. On the *Interfaces* tab, click *Add Interface*.
+
+. Navigate to the *Add Interface* form:
+.. Navigate to *Hosts* > *All Hosts*.
+.. Click *Edit* next to the host you want to edit.
+.. On the *Interfaces* tab, click *Add Interface*.
```

Comment:
```suggestion
. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
. Click *Edit* next to the host you want to edit.
. On the *Interfaces* tab, click *Add Interface*.
```

We generally try to avoid nested lists. Is that something you want to introduce on purpose? I am not against it in generally, but I like sticking to one style/to conventions. Maybe one of your colleagues can weigh in. I am OK either way and appreciate the effort.